src/code.sh

```
@@ -81,150 +44,326 @@ _trim_fastq_endings () {
+       : '''
+       Run FusionInspector per fusion list
+
+       This function will be launched as a sub job within the scope of
+       the main FusionInspector job, it sets up the job environment, runs the
+       FusionInspector command and uploads output to the parent job container to
+       then continue create the combined files and html report.
+       '''
+       set -exo pipefail
+
+       # set frequency of instance usage in logs to 30 seconds
+       kill $(ps aux | grep pcp-dstat | head -n1 | awk '{print $2}')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)